my-react-app/src/app/screens/Game/index.js

```diff
@@ -59,6 +60,14 @@ const mapDispatchToProps = dispatch => ({
   }
 });
 
+Game.propTypes = {
+  history: PropTypes.arrayOf(PropTypes.object),
```

Comment:
All of these are `isRequired`. If you didn´t pass any one of the props, the component would not work correctly.

Reply:
We never use `PropTypes.object`. In this case, you have a 
```
PropTypes.arrayOf(
 PropTypes.shape(
  {squares: PropTypes.arrayOf(PropTypes.string)}
 )
).isRequired
```